src/app/chart/values.yaml

```
@@ -8,7 +8,7 @@ checkout:
   fullnameOverride: checkout
   retail:
     checkout:
-      endpoints:
+      endpoints: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)